Gemfile.lock

```diff
@@ -13,65 +13,97 @@ GIT
   specs:
     geocoder (1.3.7)
 
+GIT
+  remote: git://github.com/wjordan/ims-lti.git
+  revision: 53e45b3f5ca7cf2b7d3654572b02628c508ceef0
+  ref: oauth_051
+  specs:
+    ims-lti (1.1.12)
+      builder
+      oauth (>= 0.4.5)
+
+GIT
+  remote: git://github.com/wjordan/konacha.git
+  revision: 228851c0daec8a32c9cfa5e348dadc809bd70329
+  ref: rails-5
+  specs:
+    konacha (4.0.0)
+      actionpack (>= 4.1, <= 5.0.0)
+      capybara
+      colorize
+      railties (>= 4.1, <= 5.0.0)
+      sprockets (>= 2, < 4)
+      sprockets-rails (>= 2, < 4)
+      tilt
+
+GIT
+  remote: git://github.com/wjordan/omniauth-clever.git
+  revision: ee8020834ef922f952cc217bb19fbfea5b1b675b
+  specs:
+    omniauth-clever (1.2.1)
+      omniauth-oauth2 (>= 1.1.1)
```

Comment:
yes- i'll add comments linking to the relevant pending GitHub issues/PRs for the forked gems i've added.